apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -403,11 +418,14 @@ private boolean checkPreValidationProperties(DataSetManifest manifest) throws Ex
       return true;
     }
 
-    // instantiate a pre-validation interface; in this case, CcwRifLoadPreValidateSynthea
+    /** we are processing Synthea data...setup a pre-validation interface. */
     CcwRifLoadPreValidateInterface preValInterface = new CcwRifLoadPreValidateSynthea();
     // initialize the interface with the appState
     preValInterface.init(appState);
+
     // perform whatever vaidation is appropriate
+    LOGGER.info(
+        "Synthea pre-validation being performed by: {}...", preValInterface.getClass().getName());
```

Comment:
Do we still need this info?

Reply:
I would say yes; this log message was actually in the original; the only difference is rather than hard-coding the class name in the log message, it is derived. But the important thing is to log a message that pre-validation is going to be performed.